qemu/tests/cfg/virtio_serial_file_transfer.cfg

```diff
@@ -30,14 +30,23 @@
                 - from_guest_to_host:
                     file_sender = guest
         - boot_with_multiple_virtserialports:
+            type = boot_N_M_virtserialports
+            only unix_socket
+            filesize = 512
             variants:
                 - 1_N:
-                    only unix_socket
-                    type = boot_1_N_virtserialports
                     driver_name = vioser
```

Comment:
a little issue, driver_name is defined in Line15

Reply:
thanks and update.